content/applications/general/email_communication/email_servers.rst

```diff
@@ -187,6 +187,36 @@ By default, inbound messages are fetched every 5 minutes in Odoo on-premise.
    :menuselection:`Settings --> Technical --> Automation --> Scheduled Actions` and look for
    :guilabel:`Mail: Fetchmail Service`.
 
+Set up different outgoing email servers for a multi-company environment
+=======================================================================
+
+The "From Filter" allows for the use of a specific outgoing email server depending on the "From"
+email address that Odoo is sending on behalf of. When an email is sent from Odoo and the "From
+Filter" is configured, then the email server will be chosen as follows:
+
+#. Odoo searches for an email server having the same "From Filter" as the "From" email address
+   defined in the outgoing email. For example, if the "From" email address is `test\@example.com`,
+   only the email servers that have the "From Filter" equal to `test\@example.com` will be returned.
+#. If no email servers are found, then Odoo searches for an email server that has the same domain
+   in the "From Filter" as the "From" email address of the outgoing email. For example, if the
+   "From" email address is `test\@example.com`, only the email servers that have the "From Filter"
+   equal to `example.com` will be returned.
+
+If no email servers are found after checking for the domain, then Odoo will return all email
+servers that do not have any "From Filter" set.
+
+Should this previous query return no results, then Odoo performs a search for an email server
+using the system parameter: `mail.default.from`. First, the email address listed will attempt
+to match an email server and then the domain will attempt a match.
+
+If no email server is found then Odoo will return the first outgoing email server (sorted by
+priority).
```

Comment:
Line broke too early on line 214.

As well, here's an optional suggestion:
Would connect that last paragraph to the main body preceding it since it's the last step in a sequence of operations... it seemed odd that paragraph would be on it's own.

```suggestion
Should this previous query return no results, then Odoo performs a search for an email server using
the system parameter: `mail.default.from`. First, the email address listed will attempt to match an
email server and then the domain will attempt a match. If no email server is found then Odoo will
return the first outgoing email server (sorted by priority).
```